thrust/thrust/detail/static_assert.h

```diff
@@ -28,34 +28,31 @@
 #include <thrust/detail/config.h>
 
 #if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
-#  pragma GCC system_header
```

Comment:
Looks like you are not using the right formatter. Please ensure that you either develop in one of our dev containers or have at least clang-format-17 installed

Reply:
I was using clang-format 16, will switch to 17 then and check formatting of other files

Reply:
Generally we strongly suggest to use the devcontainers. There everything should be setup properly